sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/internal/SensorContextTester.java

```diff
@@ -117,6 +133,15 @@ public void setActiveRules(ActiveRules activeRules) {
   }
 
   @Override
+  public SonarQubeVersion getSonarQubeVersion() {
```

Comment:
javadoc should mention that the default value is the version of API dependency and it can be overridden with `setSonarQubeVersion()`